fontforge/sfd.c

```diff
@@ -1019,12 +1029,13 @@ void SFDDumpUndo(FILE *sfd,SplineChar *sc,Undoes *u, const char* keyPrefix, int
 static void SFDDumpImage(FILE *sfd,ImageList *img) {
     GImage *image = img->image;
     struct _GImage *base = image->list_len==0?image->u.image:image->u.images[0];
+
```

Comment:
Once again, I'm sorry to nitpick, but @jtanx already got all of the substantive issues. The old organization seemed to make sense. Declarations together and then actions.